tests/com.espressif.idf.ui.test/src/com/espressif/idf/ui/test/operations/ProjectTestOperations.java

```diff
@@ -573,23 +575,20 @@ public static void launchCommandUsingContextMenu(String projectName, SWTWorkbenc
 			projectItem.select();
 			projectItem.contextMenu(contextMenuLabel).click();
 		}
-		try
-		{
-			Thread.sleep(2000);
-		}
-		catch (InterruptedException e)
-		{
-			logger.error(e.getMessage(), e);
-		}
-
+		WaitUtils.waitForJobs();
 	}
 
-	public static void waitForProjectClean(SWTWorkbenchBot bot) throws IOException
+	public static void findInConsole(SWTWorkbenchBot bot, String consoleName, String findText) throws IOException
 	{
-		SWTBotView consoleView = viewConsole("Espressif IDF Tools Console", bot);
+		SWTBotView consoleView = viewConsole(consoleName, bot);
 		consoleView.show();
 		consoleView.setFocus();
-		TestWidgetWaitUtility.waitUntilViewContains(bot, "Done", consoleView, 0);
+		TestWidgetWaitUtility.waitUntilViewContains(bot, findText, consoleView, 3000);
+	}
+
+	public static void setTextFieldInShell(SWTWorkbenchBot bot, String shellName, String textWithLabel, String setText)
+	{
+		bot.shell(shellName).bot().textWithLabel(textWithLabel + ":").setText(setText);
```

Comment:
As we discussed this is not adding any benefit we can simply use the line from this method in the actuall place of this function's call.